libs/application/ui-shell/src/lib/ApplicationForm.tsx

```diff
@@ -71,11 +72,8 @@ const ShellWrapper: FC<{
     populateForm()
   }, [fieldsDispatch, application, form, nationalRegistryId])
 
-  if (!applicationId) {
-    return <p>Error there is no id</p>
-  }
-  if (error) {
-    return <p>{error}</p>
+  if (!applicationId || error) {
+    return <NotFound />
```

Comment:
Future PR: Would be nice to have more details for Error situations, so the user could tell someone what  error they are getting. Will need design for that type of screen.